crates/spirv-std/src/arch.rs

```diff
@@ -245,3 +245,69 @@ pub fn signed_min<T: SignedInteger>(a: T, b: T) -> T {
 pub fn signed_max<T: SignedInteger>(a: T, b: T) -> T {
     unsafe { call_glsl_op_with_ints::<_, 42>(a, b) }
 }
+
+/// Index into an array or `RuntimeArray` without bounds checking.
```

Comment:
Nit: `RuntimeArray` is not implemented here, so it shouldn't be mentioned.

Also, it'd be nice to explain _why_ this exists when `get_unchecked` exists as well.

Reply:
Oh yeah I guess we shouldn't mention the `RuntimeArray` struct, but what do you call a `&[T]` when used in SPIR-V as a storage buffer?

Reply:
I'd probably call it a slice, as that's what Rust calls it.

Reply:
Yeah the terminology is tricky here. It is a slice, but it's specifically a `storage_buffer` accessible as a slice. I guess. You can't create slices from arrays (yet) in rust-gpu and you also can't sub-slice a slice either.